app/src/main/java/org/gem/indo/dooit/views/ImageActivity.java

```diff
@@ -222,6 +232,7 @@ private void processImage(int requestCodes) {
         } catch (IOException e) {
             Toast.makeText(this, "Unable to do image rotation", Toast.LENGTH_LONG).show();
             Log.e(TAG, "Unable to create temporary downscaled image file", e);
+            crashlyticsHelper.log(this.getClass().getSimpleName()," processImage : ", "an IOException");
```

Comment:
Use `Crashlytics.logException(e)` to log the stack trace to crashlytics. Use `Crashlytics.log` to record useful state up until the crash. Catching the exception and only logging "an IOException" means we're not going to get the stacktrace.

```java
try {
    myMethodThatThrows();
} catch (Exception e) {
    Crashlytics.logException(e);
}
```

https://docs.fabric.io/android/crashlytics/caught-exceptions.html